### PR TITLE
Swapping the order of episodes on lists and loops

### PR DESCRIPTION
**Pull request by maxim-belkin**
See #496 for more info.

### CODE REVIEW

File: _episodes/05-loop.md
Comment (ldko) [inline _episodes/05-loop.md:218]: typo
```suggestion
`value` is `Darwin` and `length` is 1,
```

File: _episodes/05-loop.md
Comment (ldko) [inline _episodes/05-loop.md:226]:
```suggestion
Note that a loop variable is a variable that is being used to record progress in a loop.
```

**Review by ldko — COMMENTED** [review]
I think the swap looks good. I noticed a few minor things that we might want to change.

File: _episodes/05-loop.md
Comment (ldko) [inline _episodes/05-loop.md:37]:
```suggestion
element has a unique number associated with it --- its index. This means that
```

**Comment by maxim-belkin** [comment]
Thank you, Lauren. All good suggestions/edits.

**Review by ldko — APPROVED** [review]
This looks good to me. I see the there is a build failure, but it doesn't look specifically relevant to the changes here. `make lesson-check-all` outputs what I would expect locally. 

Thanks very much for wrapping this up @maxim-belkin ! I imagine a lot of people will be glad to have this change in the lesson. Thanks to everyone who worked on it!

**Review by annefou — APPROVED** [review]
This is excellent! I also don't know why build fails: it works well for me locally.

I am really looking forward to be teaching this "new" lesson!!! thanks @maxim-belkin

**Comment by maxim-belkin** [comment]
The build fails because of the missing `github-pages` gem in Travis. I guess the fix is to add `github-pages` to the list of Gems we install on line 10 in `.travis.yml` https://github.com/swcarpentry/python-novice-inflammation/blob/4f31efad053decd7f08c39b41d14a859ad1cc344/.travis.yml#L10
but that's not critical because the main test -- GitHub Action "Website" -- passes.

As we discussed previously, we need to announce this change. Do you think we should do it before or after merging this PR?

**Comment by ldko** [comment]
> As we discussed previously, we need to announce this change. Do you think we should do it before or after merging this PR?

On the issue [you mention](https://github.com/swcarpentry/python-novice-inflammation/issues/496#issuecomment-814606963) announcing it on the Carpentries' Discuss channel. I am not sure what that refers to :D. Will a lot of people see it there? It does make sense to give notice though, in particular because of the accounts mentioned in this [recent RFC](https://github.com/carpentries/maintainer-RFCs/issues/16). It seems to me not waiting too long to merge it though would be good, so we don't lose hold of it again. Does it also make sense to do a tag/release for the version before we merge the change?

**Comment by annefou** [comment]
It is a good idea to do a tag/release for the version before merging. For announcing it: I am not sure what is best but announcing it in the Carpentries' Discuss channel seems a good idea (I guess many people will be reached).

**Comment by maxim-belkin** [comment]
> Does it also make sense to do a tag/release for the version before we merge the change?  

My understanding is that lesson releases are coordinated across The Carpentries org(s) and are performed by The Carpentries' staff.

@fmichonneau, what are our options here?

**Comment by maxim-belkin** [comment]
> On the issue [you mention](https://github.com/swcarpentry/python-novice-inflammation/issues/496#issuecomment-814606963) announcing it on the Carpentries' Discuss channel. I am not sure what that refers to :D

I was talking abouthttps://carpentries.topicbox.com/ . For example, this is our previous Request (Call) for Contributions: https://carpentries.topicbox.com/groups/discuss/Tb9e8878a4c4c4fa8-M281b5e6b472544bacf432926/call-for-contributions-visuals-images-figures-plots-charts-etc

**Pull request merged**
